libnopegl/src/node_text.c

```diff
@@ -482,7 +482,7 @@ static int init_subdesc(struct ngl_node *node,
         .graphics      = {
             .topology     = NGLI_PRIMITIVE_TOPOLOGY_TRIANGLE_STRIP,
             .state        = *graphics_state,
-            .rt_desc      = rnode->rendertarget_desc,
+            .rt_layout      = rnode->rendertarget_layout,
```

Comment:
style: there is a bunch of small spacing alignment issues in this commit due to `rt_desc` to `rt_layout` rename